clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/servergroup/model/AzureServerGroupDescription.groovy

```diff
@@ -36,12 +36,15 @@ class AzureServerGroupDescription extends AzureResourceOpsDescription {
   String provisioningState
   String application // TODO standardize between this and appName
   String clusterName
-
-  static class AzureImage {
-    String publisher
-    String offer
-    String sku
-    String version
+  String securityGroupName
+  String subnetId
+  List<String> storageAccountNames
+  AzureScaleSetCapacity capacity
+
+  static class AzureScaleSetCapacity {
+    long min
+    long max
+    Boolean useSourceCapacity
   }
```

Comment:
These values get passed in as part of the deployment of a server group in a pipeline (see Capacity tab). There's no actual usage of these at the moment at least not until we support autoscaling (in AWS these values are used when creating an autoscaling ASG).